README.md

```diff
@@ -47,6 +48,75 @@ Simply pass this through to the gateway as `token`, instead of the usual `card`
         ])->send();
 ```
 
+### Stripe Payment Intents
+
+Stripe Payment Intents is the Stripe's new foundational payment API. As opposed to Charges API, Payment Intents supports [Strong Customer Authentication](https://stripe.com/docs/strong-customer-authentication). It means that during the payment process, the user _might_ be redirected to an off-site page hosted by the customer's bank for authentication purposes.
+
+This plugin's implementation uses the manual Payment Intent confirmation flow, which is pretty similar to the one the Charges API uses. It shouldn't be too hard to modify your current payment flow.
+
+1) Start by [collecting the payment method details](https://stripe.com/docs/payments/payment-intents/quickstart#collect-payment-method) from the customer. Alternatively, if the customer has provided this earlier and has saved a payment method in your system, you can re-use that.
+
+2) Proceed to authorize or purchase as when using the Charges API.
+
+```php
+$paymentMethod = $_POST['paymentMethodId'];
+
+$response = $gateway->authorize([
+     'amount'                   => '10.00',
+     'currency'                 => 'USD',
+     'description'              => 'This is a test purchase transaction.',
+     'paymentMethod'            => $paymentMethod,
+     'returnUrl'                => $completePaymentUrl,
+     'confirm'                  => true,
+ ])->send();
+```
+
+* If you have a token, instead of a payment method, you can use that by setting the `token` parameter, instead of setting the `paymentMethod` parameter.
+* The `returnUrl` must point to where you would redirect every off-site gateway. This parameter is mandatory, if `confirm` is set to true.
+* If you don't set the `confirm` parameter to `true`, you will have to manually confirm the payment intent as shown below.
+
+```php
+$paymentIntentReference = $response->getPaymentIntentReference();
+
+$response = $gateway->confirm([
```

Comment:
Confirming a payment is not possible, when the user cancelled the authorization inside the stripe web view for 3ds.
The method throws an exception:

> Client error: `POST https://api.stripe.com/v1/payment_intents/..../confirm` resulted in a `400 Bad Request` response:
{
  "error": {
    "code": "payment_intent_unexpected_state",
    "doc_url": "https://stripe.com/docs/error-codes/paymen (truncated...)